utenti/inserimento.rst

```diff
@@ -0,0 +1,43 @@
+Inserimento nuovo utente
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. highlights::
+
+   Descrizione dei passi necessari per l'inserimento di un nuovo utente
+
+Accedendo al portale e navigando sul tab **Utenti**,
+potrai usare il pulsante **Aggiungi Utente**
+per invitare un nuovo utente sul portale.
+
+Nella schermata visualizzata, nella sezione **ANAGRAFICA**
+devono essere completati i seguenti campi:
+
+- *indirizzo email di lavoro*, dove sarà notificato l'invito
+- *Codice Fiscale*, da compilare con il codice fiscale dell'utente da invitare
+
+La sezione **Permessi sui siti web** permette di configurare la tipologia,
+e di conseguenza i permessi, dell'utente, che può essere:
```

Comment:
```suggestion
e i permessi relativi. Un utente di WAI può essere: 
```

Reply:
dov'è il ruolo dell'utente in sola lettura? lo specificherei meglio a parte così come fatto per quello di amministratore

Reply:
I ruoli in realtà sono due amministratore/non-amministratore.
L'amministratore ha permessi di lettura/gestione su tutti i siti (oltre a gestione utenti e siti).
I non-amministratori, invece, hanno permessi specifici sui singoli siti.
Pensi sia più chiaro così?